assets/js/modules/adsense/datastore/service.js

```diff
@@ -125,6 +126,13 @@ export const selectors = {
 		return select( STORE_NAME ).getServiceURL( { path, query } );
 	} ),
 
+	getServiceReportURL: createRegistrySelector( ( select ) => ( state, reportArgs ) => {
```

Comment:
Please add a doc block to this function, since it's a regular selector (which should be documented).